parsec/api/protocol/realm.py

```diff
@@ -62,6 +63,15 @@ class RealmStatusRepSchema(BaseRepSchema):
 
 realm_status_serializer = CmdSerializer(RealmStatusReqSchema, RealmStatusRepSchema)
 
+class RealmStatsReqSchema(BaseReqSchema):
+    realm_id = fields.UUID(required=True)
+
+
+class RealmStatsRepSchema(BaseRepSchema):
+    data_size = fields.Integer(required=True)
```

Comment:
would be better to name this `blocks_size` (so we can easily tell how this is calculated, and we can add a `vlobs_size` latter on)